docs/how-to-guides/creating-grc20.md

```diff
@@ -142,33 +147,36 @@ func Render(path string) string {
 	case path == "":
 		return mytoken.RenderHome()
 	case c == 2 && parts[0] == "balance":
-		owner := pusers.AddressOrName(parts[1])
-		balance, _ := mytoken.BalanceOf(users.Resolve(owner))
+		owner := std.Address(parts[1])
+		balance, _ := mytoken.BalanceOf(owner)
 		return ufmt.Sprintf("%d\n", balance)
 	default:
 		return "404\n"
 	}
 }
-
-func assertIsAdmin(address std.Address) {
-	if address != admin {
-		panic("restricted access")
-	}
-}
 ```
 
 Detailing what is happening in the above code:
 - Calling the `TotalSupply` method would return the total number of tokens minted.
 - Calling the `BalanceOf` method would return the total balance of an account.
-- Calling the `Allowance` method would set an account as an allowed spender to serve on behalf of the owner.
-- Calling the `transfer` method would transfer a configurable amount of token from the calling account to another account, either owned or unowned.
-- Calling the `Approve` method would approve a calling account to spend a configurable amount of token on behalf of the token owner.
-- Calling the `TransferFrom` method would transfer a configurable amount of token from an account that granted approval to another account, either owned or unowned.
-- Calling the `Mint` method would create a configurable number of tokens by the administrator.
-- Calling the `Burn` method would destroy a configurable number of tokens by the administrator.
-- Calling the `Render` method would return a user's `balance` as a formatted string. Learn more about the `Render`
+- Calling the `Allowance` method would set an account as an allowed spender to
```

Comment:
(for a future update) It would be beneficial to implement line highlighting, which would distinctly mark each method reference alongside the corresponding method within the code block above.